nss-day-cohort-39/tabloid-full-stack-del_rios
# Post Tags

### Details
- Users can add, edit, or remove tags to be added to posts
- Users can add or remove tags to their post in the post details

### How to Test
- Try adding, editing, and deleting tags from tag management
- Try adding tags to a post by managing tags in your post's details, ensuring that they show on the post
- Try removing tags from your post
- Make sure you can't add tags to posts that do not belong to you
